### PR TITLE
chore: rename "Library" to "External Library" in system settings

### PR DESCRIPTION
**Pull request by declan8010**
This is intended to assist with any confusion regarding standard libraries.

### CODE REVIEW

File: web/src/routes/admin/system-settings/+page.svelte [hunk 1]
```diff
@@ -72,8 +72,8 @@
     },
     {
       item: LibrarySettings,
-      title: 'Library',
-      subtitle: 'Manage library settings',
+      title: 'External Library',
+      subtitle: 'Manage external library settings',
       key: 'library',
```
Comment (jrasm91) [inline web/src/routes/admin/system-settings/+page.svelte:77]: might as well change this to `external-library` as well.

Comment (declan8010) [reply web/src/routes/admin/system-settings/+page.svelte]: Updated 👍

**Comment by aviv926** [comment]
Can you change that as well?
Assuming there is no problem that it will be together with this PR @jrasm91.

image

**Comment by declan8010** [comment]
Actioned 👍

**Review by jrasm91 — APPROVED** [review]

**Pull request merged**